disassemblers/ddisasm/ddisasmBBorRef.py

```diff
@@ -0,0 +1,305 @@
+import logging
+import optparse
```

Comment:
`optparse` is deprecated since python 3.2, why use it? I think we should be using `argparse`

Reply:
Fixed.